src/lib/ThumbnailsSidebar.js

```diff
@@ -120,9 +174,16 @@ class ThumbnailsSidebar {
      */
     createPlaceholderThumbnail(itemIndex) {
         const thumbnailEl = document.createElement('button');
-        thumbnailEl.className = 'bp-thumbnail';
+        const pageNum = itemIndex + 1;
+
+        thumbnailEl.className = CLASS_BOX_PREVIEW_THUMBNAIL;
         thumbnailEl.setAttribute('type', 'button');
-        thumbnailEl.appendChild(this.createPageNumber(itemIndex + 1));
+        thumbnailEl.dataset.bpPageNum = pageNum;
+        thumbnailEl.appendChild(this.createPageNumber(pageNum));
+
+        if (pageNum === this.currentPage) {
```

Comment:
I feel like the setting of selected thumbnail should be handled by the caller, not inside this method itself. Do we call this in many places?

Reply:
let me see how to move this out

Reply:
As we discussed, I'm going to leave this in since the performance cost should be minimal and what this does is allow the placeholder thumbnail to have the selected class & appearance while you're scrolling through